text/text.md

```diff
@@ -1620,55 +1620,55 @@ The second greatest show on earth goes to New Orleans
-did not know already that all that Noah stuff was a big steaming pile
+didn't know already that all that Noah stuff was a big steaming pile
 of elephant dung, but if you put lions and horses and bearded women
 and whatnots inside a steamer they're gonna get mighty nervous, I tell
 ya. So no way Noah would have made it for forty days. Unless he got him a
-circus manager like we did. Friend of saying platitudes, but a good organizer
+circus manager like we did. Fond of saying platitudes, but a good organizer
 all the same.  
 
 Yes, I would have preferred an airship. The circus comisario was on its
 way to New Orleans already. But, pray tell, how the fuck do you expect
 to put an elephant in an airship? You did it once? Really? Why didn't
 you stick a couple of wings and a rocket up its butt and made it fly?
 No way, man, absolutely no way you could put the thirty-odd carriages
-in airships, you would have needed so many of them. Yes, you love
+in an airship, you would have needed many of them. Yes, you love
 airships, Commodore, that's a fact.
 
-This was my first chance to actually drive the mech. Aw, man, we are
-calling it a hoborg, but just from a couple of nights ago! Just let me
+This was my first chance to actually drive the mech. Aw, man, we were
+calling it a hoborg just a couple of nights ago! Just let me
 call it the way I've always done. You want to hear the story? As I was
 saying, you already know it has to be driven. On its own, he is barely
 able to repeatedly bump into a wall or wander aimlessly like a chicken
 whose head has just been bitten by a geek. Or a frog, yes,
 Frog-Eatin'.  Grease was the one usually driving the mech, but he was
 too busy disengaging carriages from the tractor, securing them on the
 deck and trying at the same time to put the poor beasts at ease. The
-space occupied by the mech was needed for something so I got to drive
+space occupied by the mech was needed, so I got to drive
 it for the first time. 
 
 No, it does not even come close to driving a horse and I ain't never
 piloted a car. It's a metal box with a wheel and several levers and
-knobs. I was told to never get farther than a dozen feet from from it,
+knobs. I was told to never get farther than a dozen feet from from it,     /*  not get a dozen feet from it? unclear what is meant*/ 
 this wheel for turning, this lever forward, this knob to make it
 faster. Grease gave it to me, and I swear it turned its teakettle head
 to look at me. Yes, it does that. It's not only levers and knobs and
 whatnots. I still don't know what half the driving box does, but I do
 know that the fucking mech has a mind of its own. No soul, Blackie, no
 soul, man. No man's got a soul, nobody. It's humbug the American
 capitalistas have been feeding you. So how come a fucking mechanical
-contraption have a soul if not even you have it!  But, boy, does it
+contraption can have a soul if not even you have it!  But, boy, does it
```

Comment:
*how come it have a soul*, is what I was aiming for here. "Come" is colloquial for "can"